dcos/httpclient.go

```diff
@@ -0,0 +1,69 @@
+package dcos
+
+import (
+	"net"
+	"net/http"
+	"time"
+)
+
+const (
+	userAgent = ClientName + " - " + Version
+)
+
+type DCOSTransport struct {
```

Comment:
Since it's in the `dcos` package I think we can get rid of the `DCOS` and just call it something like `DefaultTransport`, which would then be `dcos.DefaultTransport`  to users (as opposed to `dcos.DCOSTransport`)

Reply:
Will do thanks